db/migrate/20240408093541_change_default_accent_and_bim_primary_color.rb

```diff
@@ -0,0 +1,32 @@
+class ChangeDefaultAccentAndBimPrimaryColor < ActiveRecord::Migration[7.1]
+  class MigrationDesignColor < ApplicationRecord
+    self.table_name = "design_colors"
+  end
+
+  def up
+    # The default Bim value was forgotten in previous migrations
+    primary_button_color = MigrationDesignColor.find_by(variable: "primary-button-color")
+    if primary_button_color&.hexcode == OpenProject::CustomStyles::ColorThemes::DEPRECATED_BIM_ALTERNATIVE_COLOR
+      primary_button_color.update(hexcode: OpenProject::CustomStyles::ColorThemes::PRIMER_PRIMARY_BUTTON_COLOR)
+    end
```

Comment:
The code for this could be simplified to

```
MigrationDesignColor
  .where(variable: "primary-button-color", hexcode: OpenProject::CustomStyles::ColorThemes::DEPRECATED_BIM_ALTERNATIVE_COLOR)
  .update_all(hexcode: OpenProject::CustomStyles::ColorThemes::PRIMER_PRIMARY_BUTTON_COLOR)
```

It would also run faster since it does not instantiate the record.